src/http_server.cc

```diff
@@ -331,140 +369,140 @@ ReadDataFromJsonHelper(
 {
   // FIXME should invert loop and switch so don't have to do a switch
```

Comment:
should we delete this comment?

Reply:
will update comment, here doesn't really change the handling to be address (still switch on each element even though all elements are the same data type)